README.rst

```diff
@@ -0,0 +1,35 @@
+==============
+ Z3C template
```

Comment:
Maybe this line should repeat the package name.

Reply:
I didn't adjust the text, just the over/underlines to make them consistent and support the extra level of indentation I needed. I can alter this text.

Reply:
`Z3C` does not exist as an acronym anywhere else but maybe the English language enforces it to write this in upper case here because it is a headline and `Z3c` would look even more ugly.

Reply:
I've always understood Z3C to be an abbreviation for Zope 3 Community, and yeah, abbreviations are upper-cased in English.